src/System.Runtime.CompilerServices.VisualC/ref/System.Runtime.CompilerServices.VisualC.cs

```diff
@@ -32,6 +32,9 @@ public partial class CallConvThiscall
     public static partial class IsBoxed
     {
     }
+    public static partial class IsConst
```

Comment:
@ramarag IsConst is already defined in System.Runtime. Please remove it from here as it causes conflicts.